src/core/Plugin_factoids.ml

```diff
@@ -175,15 +192,15 @@ let search tokens (fcs:t): string list =
 let random (fcs:t): string =
   let l = StrMap.to_list fcs in
   match l with
-    | [] -> ""
-    | _ ->
-      let _, fact = Rand_distrib.uniform l |> Rand_distrib.run in
-      let msg_val = match fact.value with
-        | StrList [] -> assert false
-        | StrList l -> Rand_distrib.uniform l |> Rand_distrib.run
-        | Int i -> string_of_int i
-      in
-      Printf.sprintf "!%s: %s" fact.key msg_val
+  | [] -> ""
```

Comment:
For some reason, ocp-indent changed the identation on the lower block - I was also surprised. I'll restore the old indentation.